doc/classes/VisualServer.xml

```diff
@@ -1680,6 +1680,7 @@
 			</argument>
 			<description>
 				Returns [code]true[/code] if the OS supports a certain feature. Features might be [code]s3tc[/code], [code]etc[/code], [code]etc2[/code] and [code]pvrtc[/code].
+				When rendering with GLES2, returns [code]true[/code] with [code]skinning_fallback[/code] in case the hardware doesn't support the default GPU skinning process.
```

Comment:
The previous line lists valid features, but it doesn't include `skinning_fallback`.